db/seeds.rb

```diff
@@ -16,10 +16,12 @@
 
 OX_MEMBER = Member.find_by(name: OX)
 unless User.exists?(member_id: OX_MEMBER.id)
-  account = OpenStax::Accounts::Account.create!(
-    uuid: SecureRandom.uuid
+  account = ::OpenStax::Accounts::FindOrCreateAccount.call(
+    email: 'support@openstax.org', username: 'openstax', password: 'password'
+  ).outputs.account
+  User.create!(
+    member_id: OX_MEMBER.id, account_id: account.id, role: 'power_user'
```

Comment:
Can you make two users? One with username `ox_power_user` and another with `ox_user` usernames?  (power user and normal user).  Can probably just use bogus email accounts since this is just for dev.  Oh on that note please do enclose this in `if Rails.env.development?` so that it doesn't accidentally get triggered on prod.